src/Altinn.App.Core/Features/Telemetry.Instance.cs

```diff
@@ -0,0 +1,144 @@
+    {
+        var activity = ActivitySource.StartActivity(TraceNameCreate);
+        activity?.SetTag(InstanceLabels.Type, type.ToStringFast());
+        activity?.SetTag(Labels.InstanceGuid, instanceGuid);
+        activity?.SetTag(Labels.InstanceOwnerPartyId, instanceOwnerPartyId);
+        return activity;
+    }
+
+    internal Activity? StartUpdateProcessActivity(InstanceType type)
+    {
+        var activity = ActivitySource.StartActivity(TraceNameProcess);
+        activity?.SetTag(InstanceLabels.Type, type.ToStringFast());
+        return activity;
+    }
+
+    internal Activity? StartCompleteConfirmationActivity(InstanceType type, Guid instanceGuid)
+    {
+        var activity = ActivitySource.StartActivity(TraceNameConfirmation);
+        activity?.SetTag(InstanceLabels.Type, type.ToStringFast());
+        activity?.SetTag(Labels.InstanceGuid, instanceGuid);
+        return activity;
+    }
+
+    internal Activity? StartUpdateReadStatusActivity(InstanceType type, Guid instanceGuid, int instanceOwnerPartyId)
+    {
+        var activity = ActivitySource.StartActivity(TraceNameReadStatus);
+        activity?.SetTag(InstanceLabels.Type, type.ToStringFast());
+        activity?.SetTag(Labels.InstanceGuid, instanceGuid);
+        activity?.SetTag(Labels.InstanceOwnerPartyId, instanceOwnerPartyId);
+        return activity;
+    }
+
+    internal Activity? StartUpdateSubStatusActivity(InstanceType type, Guid instanceGuid, int instanceOwnerPartyId)
+    {
+        var activity = ActivitySource.StartActivity(TraceNameSubStatus);
+        activity?.SetTag(InstanceLabels.Type, type.ToStringFast());
+        activity?.SetTag(Labels.InstanceGuid, instanceGuid);
+        activity?.SetTag(Labels.InstanceOwnerPartyId, instanceOwnerPartyId);
+        return activity;
+    }
+
+    internal Activity? StartUpdatePresentationTextActivity(InstanceType type, Guid instanceGuid, int instanceOwnerPartyId)
+    {
+        var activity = ActivitySource.StartActivity(TraceNamePresentationText);
+        activity?.SetTag(InstanceLabels.Type, type.ToStringFast());
+        activity?.SetTag(Labels.InstanceGuid, instanceGuid);
+        activity?.SetTag(Labels.InstanceOwnerPartyId, instanceOwnerPartyId);
+        return activity;
+    }
+
+    internal Activity? StartUpdateDataValuesActivity(InstanceType type, Guid instanceGuid, int instanceOwnerPartyId)
+    {
+        var activity = ActivitySource.StartActivity(TraceNameDataValues);
+        activity?.SetTag(InstanceLabels.Type, type.ToStringFast());
+        activity?.SetTag(Labels.InstanceGuid, instanceGuid);
+        activity?.SetTag(Labels.InstanceOwnerPartyId, instanceOwnerPartyId);
+        return activity;
+    }
+
+    internal static class Instance
+    {
+        private static readonly string _prefix = "Instance";
+
+        internal static readonly string TraceNameGet = $"{_prefix}.Get";
```

Comment:
I think all these work with `const`
```suggestion
        internal const string TraceNameGet = $"{_prefix}.Get";
```

Reply:
Agreed